### ISSUE TITLE
Mark up and move Categories side bar (news page)

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#851]
**Page 161 in DAC report**

**Issue from DAC report:**
The ‘Posts by Category’ side navigation region (see updates image) is not marked-up. As it is placed after the rest of the main page content within the HTML, screen reader users will encounter it after the rest of the page is relayed. The context of its visual positioning alongside the main content and at the top of the page is not presented to screen reader users.

**Solution from DAC report:**
Please ensure the sidebar is marked-up within an ‘aside’ region. In addition, it should be included above the main page content to ensure that users encounter it in page sequence prior to the main content to which it relates.

**Example from DAC report:**

```
<aside id="sidebar">
    <h4>Posts by Category</h4>
    <ul> …</ul>
</aside>
<main id="main-content" class="container newslist">…</main>
```

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin At first glance I don't agree with moving the markup for the post-categories above the main section. For sighted users, the post categories appear in the right sidebar. I've always thought of right sidebars as being "after" the main section. So unless my thinking is wrong there, sighted users are getting the same experience as screen-reader users, with the sidebar coming after the main section in the markup. And in fact, from a user experience perspective, we wouldn't want to present links to related content _before_ the user has read the content on the page. But maybe it's worth double-checking with DAC in case I'm missing something.

In the meantime I've created a PR for the `aside` suggestion: https://github.com/open-sdg/open-sdg/pull/890
